projects/ui/src/lib/components/po-table/po-table-cell-template/po-table-cell-template.directive.ts

```diff
@@ -0,0 +1,74 @@
+import { Directive, TemplateRef } from '@angular/core';
+/**
+ * @usedBy PoTableComponent
+ *
+ * @description
+ *
+ * Esta diretiva permite que seja possível alterar o conteúdo das celúlas de uma coluna, para que os valores possam ser exibidos
+ * de acordo com a necessidade do usuário.
```

Comment:
esse trecho está igual a documentação da outra diretiva, poderia deixar mais específica a diferença entre elas já nesse início da documentação.

Reply:
Então, esta  igual porque os 2 componentes fazem a mesma coisa, a difrença esta no exemplo de utilização. Foi o que pensei no momento, mas se tiver uma sugestão? Ai eu altero.